cvat/apps/engine/serializers.py

```diff
@@ -795,6 +803,12 @@ def validate(self, attrs):
             and attrs['start_frame'] > attrs['stop_frame']:
             raise serializers.ValidationError('Stop frame must be more or equal start frame')
 
+        if (
+            (server_files := attrs.get('server_files'))
+            and attrs.get('cloud_storage_id')
+            and len(list(filter(lambda x: not x['file'].endswith('.jsonl'), server_files))) > settings.CLOUD_STORAGE_MAX_FILES_COUNT
```

Comment:
```suggestion
            and sum(1 for f in server_files if f['file'].endswith('.jsonl'))
            and len(list(filter(lambda x: not x['file'].endswith('.jsonl'), server_files))) > settings.CLOUD_STORAGE_MAX_FILES_COUNT
```

Reply:
Thanks, changed to `sum(1 for f in server_files if not f['file'].endswith('.jsonl'))`